Backend/NUnit_Collections/NUnit_Collection_Tests/Collections.cs

```diff
@@ -78,7 +78,7 @@ public void Test_Collection_AddRange()
         public void Test_Collection_InsertAt()
```

Comment:
Rename this test to Test_Collection_InsertAt_Valid_Index

Reply:
Like I sad on the meeting I've done my tests by the presentation examples.